packages/aws-cdk-lib/aws-dynamodb/lib/table.ts

```diff
@@ -1788,6 +1798,25 @@ export class Table extends TableBase {
       },
     };
   }
+
+  private renderPointInTimeRecoverySpecification(
+    pointInTimeRecovery?: boolean,
+    recoveryPeriodInDays?: number,
+  ): CfnTable.PointInTimeRecoverySpecificationProperty | undefined {
+    if (pointInTimeRecovery == null && recoveryPeriodInDays === undefined) return undefined;
```

Comment:
I'd prefer to refactor this to the below suggestion but conditional on thoughts to [this thread](https://github.com/aws/aws-cdk/pull/33010/files#r1921117747).

```suggestion
    if (pointInTimeRecovery == null && recoveryPeriodInDays === undefined) return undefined;
```